models/Spot.js

```diff
@@ -3,20 +3,27 @@ const utils = require('../utils');
 const attributes = ['id', 'open', 'challengedCount'];
 
 module.exports = class Spot {
-  static getAttributes() {
+
+  constructor(id, open, challengedCount) {
+    this._id = id;
+    this._open = open;
+    this._challengedCount = challengedCount;
+  }
+
+  static get attributes() {
     return attributes;
   }
 
-  static getIdName() {
+  static get idName() {
     return 'id';
   }
 
-  static getTableName() {
+  static get tableName() {
     return 'spots';
   }
 
   // true is open, false is closed
-  setOpenClose(spotId, open) {
+  static setOpenClose(spotId, open) {
```

Comment:
Part of me wants this to me an instance method, so you wouldn't have to pass in the ID. Rough sketch below:

```js
const spot = Spot.findById(123);
spot.open = true;
spot.save();
```

Definitely not a big deal, but I feel like it might encapsulate things a bit more. (Also probably not worth changing before tomorrow.)